internal/flink/command_compute_pool_delete.go

```diff
@@ -46,7 +49,7 @@ func (c *command) computePoolDelete(cmd *cobra.Command, args []string) error {
 		return err == nil
 	}
 
-	if err := deletion.ValidateAndConfirmDeletion(cmd, args, existenceFunc, resource.FlinkComputePool, computePool.Spec.GetDisplayName()); err != nil {
+	if err := validateAndConfirmComputePoolDeletion(cmd, args, existenceFunc, resource.FlinkComputePool, computePool.Spec.GetDisplayName()); err != nil {
```

Comment:
To be more consistent w/ other CLI resources that have extra delete information, can we add the extra information (about 30 days in list history & permanent deletion) as a message after the delete operation? Instead of giving the information before in this new validation function.

Like in kafka/command_broker_delete.go which calls `deletion.DeleteWithoutMessage` and then outputs a custom post-delete message.

Reply:
This is an irreversible operation and this could lead to people stopping production flink jobs by accident. That's why I think it's actually better to include the info in the warning before the user deletion the pool it.

Also, the feature was designed in a product one pager, and it would be up to product to decide @kasher-confluent

Reply:
I do agree with @Guttz. Deleting a compute pool can have consequences on the customer's workload. Thus, it is important to inform them about what is expected to happen when they delete a compute pool. We do the same for the UI as well. Meaning, show them the below popup before deletion to confirm. 

<img width="558" alt="Screenshot 2024-04-25 at 07 47 48" src="https://github.com/confluentinc/cli/assets/126028752/dc5e192d-2d2b-4048-a4aa-d10a6483f883">

Would still recommend showing the confirmation prompt before deletion.

Reply:
Ok, this seems fine then.